docs/DeveloperGuide.adoc

```diff
@@ -150,97 +185,132 @@ The `Storage` component,
 Classes used by multiple components are in the `seedu.addressbook.commons` package.
 
 == Implementation
-
 This section describes some noteworthy details on how certain features are implemented.
 
-// tag::undoredo[]
-=== [Proposed] Undo/Redo feature
-==== Proposed Implementation
+=== Add Interviewee/Interviewer feature
+The add command is facilitated by the `AddCommandParser` class. It extends `AddCommandParser` by allowing it to parse
+prefixes specific to our Scheduler app, namely:
 
-The undo/redo mechanism is facilitated by `VersionedAddressBook`.
-It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`.
-Additionally, it implements the following operations:
+* r/ROLE
+* f/FACULTY
+* ep/PERSONAL_EMAIL and ew/NUS_WORK_EMAIL
+* y/YEAR_OF_STUDY
+* d/DEPARTMENT
+* s/SLOT
 
-* `VersionedAddressBook#commit()` -- Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` -- Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` -- Restores a previously undone address book state from its history.
+This command also requires access to the Model package, where the AddressBook implementation lives, as well as the
```

Comment:
Not sure if the references to AddressBook are intended, but perhaps it's better to leave them out because it could be confusing?